collector/systemd_linux.go

```diff
@@ -42,15 +42,15 @@ const (
 )
 
 var (
-	unitIncludeSet bool
-	unitInclude    = kingpin.Flag("collector.systemd.unit-include", "Regexp of systemd units to include. Units must both match include and not match exclude to be included.").Default(".+").PreAction(func(c *kingpin.ParseContext) error {
-		unitIncludeSet = true
+	systemdIncludeSet bool
+	systemdInclude    = kingpin.Flag("collector.systemd.unit-include", "Regexp of systemd units to include. Units must both match include and not match exclude to be included.").Default(".+").PreAction(func(c *kingpin.ParseContext) error {
```

Comment:
I would argue that this should be `systemdUnitInclude` to be more specific.

But either way, can we open this as a separate PR?

Reply:
Fair enough, removing from this PR.